scripts/port-changes.js

```diff
@@ -7,165 +7,216 @@
  * to the new branch.
  */
 
-const shell = require('shelljs');
+const shell = require("shelljs");
 
 const PR_REGEX = new RegExp(/(\(#\d+\))(\s+\(#\d+\))*$/);
 const COMMIT_REGEX = new RegExp(/^([\da-zA-Z]+)/);
 
-const PR_NUM = 'PR_NUM';
-const COMMIT = 'COMMIT';
-const MESSAGE = 'MESSAGE';
+const PR_NUM = "PR_NUM";
```

Comment:
was there a change on the eslint rules ?

Reply:
It didn't look like it, this is what the formatter gives me now though. We can sync about this after standup.